pychromecast/socket_client.py

```diff
@@ -268,9 +268,7 @@ def mdns_backoff(
             retry["delay"] = min(retry["delay"] * 2, 300)
             retries[service] = retry
 
-        while not self.stop.is_set() and (
-            tries is None or tries > 0
-        ):  # pylint:disable=too-many-nested-blocks
+        while not self.stop.is_set() and (tries is None or tries > 0):  # pylint:disable=too-many-nested-blocks
```

Comment:
Can we use `disable-next` and put the comment on the line above, to avoid having a long line?